ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -772,6 +776,69 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'TOP Latent Defect Tower',
+      type: 'StartsUsing',
+      netRegex: { id: '7B6F', source: 'Omega', capture: false },
+      infoText: (data, _matches, output) => {
+        const myColor = data.bugRot[data.me];
+        if (data.defamationColor === myColor)
+          return output.colorTowerDefamation!({ color: myColor });
+        else if (myColor)
+          return output.colorTower!({ color: myColor });
+      },
+      run: (data) => {
+        data.bugRot = {};
+        data.latentDefectCount = data.latentDefectCount + 1;
+      },
+      outputStrings: {
+        colorTower: {
+          en: '${color} Tower',
```

Comment:
```suggestion
          en: '${color} Tower Stack',
```

Reply:
Should this have said Soak instead of Stack?

Reply:
Forgot to mention, the callout only goes to those with the rot debuff. I was think a trigger based on the patch tether + defamation would be for the others but it would be similar output.

Reply:
I might be wrong, I haven't tested this in game, only in raidemulator. But it is true I just realized the stack could be an indicator to be near the blue tether which is needed.